bedrock/firefox/templates/firefox/new/basic/base_download.html

```diff
@@ -59,14 +60,15 @@
   <div class="mzp-c-notification-bar mzp-t-warning" data-nosnippet="true">{{ ftl('firefox-new-mozilla-no-longer-provides', fallback='firefox-new-youre-using-an-insecure-outdated', url='https://support.mozilla.org/kb/end-support-windows-xp-and-vista') }}</div>
   <![endif]-->
 
-  {% call hero(
-    title=hero_title,
-    class='mzp-has-image hero-firefox t-narrow',
-    include_cta=True,
-    image_url=hero_img,
-    heading_level=1
-  ) %}
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md t-split-hero',
+    media_class='mzp-l-split-media-overflow',
+    media_after=False,
+    image=resp_img(hero_img)
```

Comment:
Do we know if `hero_img`  is always a URL? If so, it's fine to expand out the resp_img call to include the optional class attribute (needed for split overflow styling)
```suggestion
    image=resp_img(url=hero_img, optional_attributes={ 'class': 'mzp-c-split-media-asset'})
```